Plugin/src/SofaPython3/config.h

```diff
@@ -41,8 +41,7 @@
 
 // define own pybind version macro, much easier to test/read
 #define PYBIND11_SOFA_VERSION (PYBIND11_VERSION_MAJOR * 10000 \
-    + PYBIND11_VERSION_MINOR * 100 \
-    + PYBIND11_VERSION_PATCH)
+    + PYBIND11_VERSION_MINOR * 100)
 // pybind11 already bind the attributeError starting from 2.8.1 version.
 // so if the version is >= 2.8.1, macro does nothing, otherwise bind attribute_error
 #if PYBIND11_SOFA_VERSION >= 20801
```

Comment:
Actually just looking at this code makes me think that the patch version is useful.

Reply:
Another solution could be to use CMake generate to set `PYBIND11_SOFA_VERSION`. It implies to change this config.h into config.h.in
I can do it if you want.

Reply:
Much better in my opinion. I don't know how I missed the comment, right between the macro declaration and the version condition check....

I configured the macro with cmake in the last commit